src/table/table.scss

```diff
@@ -117,12 +119,15 @@
 }
 
 @mixin iui-table-body {
-  overflow-x: hidden;
+  overflow-x: auto;
+  overflow-x: overlay;
   overflow-y: scroll;
   overflow-y: overlay;
```

Comment:
These can be combined into `overflow: auto; overflow: overlay;`

Reply:
Changed